File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/Utenlandsopphold.java
package no.nav.domain.pensjon.regler.grunnlag;

import no.nav.domain.pensjon.regler.kode.LandCti;

import java.io.Serializable;
import java.util.Date;

public class Utenlandsopphold implements Serializable {

    /**
     * Fra og med dato
     */
    private Date fom;
    /**
     * Til og med dato
     */
    private Date tom;
    /**
     * Landet hvor oppholdet har funnet sted
     */
    private LandCti land;
    /**
     * Hvilken pensjonsordning som skal brukes
     */
    private String pensjonsordning;
    /**
     * Har personen bodd i utlandet
     */
    private boolean bodd;
    /**
     * Har personen arbeidet i utlandet
     */
    private boolean arbeidet;

}

File: src/main/java/no/nav/pensjon/regler/domain/grunnlag/Utenlandsopphold.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.kode.LandCti
import java.io.Serializable
import java.util.*

class Utenlandsopphold : Serializable {
    /**
     * Fra og med dato
     */
    var fom: Date? = null

    /**
     * Til og med dato
     */
    var tom: Date? = null

    /**
     * Landet hvor oppholdet har funnet sted
     */
    var land: LandCti? = null

    /**
     * Hvilken pensjonsordning som skal brukes
     */
    var pensjonsordning: String? = null

    /**
     * Har personen bodd i utlandet
     */
    var isBodd = false

    /**
     * Har personen arbeidet i utlandet
     */
    var isArbeidet = false

}